src/main/java/com/mixpanel/android/viewcrawler/ViewVisitor.java

```diff
@@ -204,73 +204,42 @@ public LayoutUpdateVisitor(List<Pathfinder.PathElement> path, LayoutRule args,
 
         @Override
         public void cleanup() {
-            for (Map.Entry<View, LayoutRule> original:mOriginalValues.entrySet()) {
+            // TODO find a way to optimize this.. remove this visitor and trigger a re-layout??
```

Comment:
If I understand it correctly, we are not actually changing the attributes of the views in the app, but overwrite them. So, my question is, why do we even write the original value back if the attributes are not changed? Why don't we just trigger the window refresh in some way and leave the attributes as they are?